src/toolkit.cpp

```diff
@@ -1704,7 +1705,7 @@ std::string Toolkit::GetTimesForElement(const std::string &xmlId)
     jsonxx::Object o;
 
     if (!element) {
-        LogWarning("Element '%s' not found", xmlId.c_str());
+        LogInfo("Element '%s' not found", xmlId.c_str());
```

Comment:
Why was this log level changed?

Reply:
Because it returns an empty json. To me this looks awkward. When an unknown id produces a warning I'd expect that nothing is returned. But then we need to handle it differently.

Reply:
Since you made it a warning line 1642 it would probably make sense to keep it consistent and to leave it a warning. Furthermore, since the default log level is now `warning`, then it is probably wise to have a message logged by default when the element is not found.